### PR TITLE
Add links to all header tags

### PR DESCRIPTION
**Pull request by picklelo**

### CODE REVIEW

File: pcweb/templates/docpage.py [hunk 1]
```diff
@@ -398,38 +374,28 @@ def docheader2(
...
     return rx.box(
-        *children,
-        id=id_,
+        rx.hstack(
+            rx.heading(text, _as=tag, id=id_, style=style, **props),
+            rx.icon(
+                tag="link",
+                style=icon_style,
+                color=styles.ACCENT_COLOR,
+                on_click=lambda: rx.set_clipboard(State.current_page + "#" + id_)
```
Comment (masenf) [inline pcweb/templates/docpage.py:395]: i think we should make this a real link and set the `href` too, as many people (myself included) right click these guys and "Copy Link"

**Review by masenf — CHANGES_REQUESTED** [review]

File: pcweb/templates/docpage.py (hunk 1 above)
Comment (picklelo) [reply pcweb/templates/docpage.py]: Good call, updated to use a link instead

**Review by Alek99 — APPROVED** [review]
lgtm

**Review by masenf — APPROVED** [review]
The thought the "click to copy" thing was cool too, was it not working to have both?

**Comment by picklelo** [comment]
> The thought the "click to copy" thing was cool too, was it not working to have both?

I brought this back now, so it should do both.

**Review by masenf — APPROVED** [review]

**Pull request merged**